src/components/new-request/search.vue

```diff
@@ -357,150 +564,293 @@ export default class NewSearch extends Mixins(CommonMixin) {
   }
 
-  get isInternational () {
-    return (this.location === Location.IN)
+  /** Whether selected radio button is Named Company. */
+  get isNamedCompany (): boolean {
+    return (this.selectedCompanyType === CompanyType.NAMED_COMPANY)
   }
 
-  get isPersonsName () {
-    return this.getIsPersonsName
+  get showActionButton (): boolean {
+    if (this.isConversion && !this.isAlterOnline(this.getConversionType)) return false
+    return true
   }
 
-  set isPersonsName (value) {
-    this.setIsPersonsName(value)
-  }
+  /** Whether to show "Go to COLIN" button (otherwise will show `actionNowButtonText` button). */
+  get showColinButton (): boolean {
+    if (this.showContinueInButton) return true
+    if (this.isFederal) return true
 
-  get location (): Location {
-    return this.getLocation
-  }
+    // don't show COLIN button for supported alteration entities
+    if (this.isConversion) {
+      return !this.isSupportedAlteration(this.getConversionType)
+    }
+
+    // don't show COLIN button for supported restoration entities
+    if (this.isRestoration) {
+      const supportedRestorationEntites = GetFeatureFlag('supported-restoration-entities')
+      const isRestorationEntity = supportedRestorationEntites.includes(this.entity_type_cd)
+      return !isRestorationEntity
+    }
 
-  set location (location: Location) {
-    this.setLocation(location)
+    if (this.isChangeName) {
+      const supportedChnageNameEntites = GetFeatureFlag('supported-name-change-entities')
+      const isChangeNameEntity = supportedChnageNameEntites.includes(this.entity_type_cd)
+      return !isChangeNameEntity
+    }
+
+    // don't show COLIN button for supported entities
+    const supportedEntites = GetFeatureFlag('supported-incorporation-registration-entities')
+    const isIncorporateEntity = supportedEntites.includes(this.entity_type_cd)
+    return !isIncorporateEntity
   }
 
-  get jurisdiction (): string {
-    return this.getJurisdictionCd
+  get showContinueInButton (): boolean {
+    // for now, return True because Continuation In filings are not yet implemented
+    return this.isContinuationIn
+
+    // *** FUTURE: use code below
+    // if (!this.isContinuationIn) return false
+    // const supportedContInEntites = GetFeatureFlag('supported-continuation-in-entities')
+    // const isContInEntity = supportedContInEntites.includes(this.entity_type_cd)
+    // return !isContInEntity
   }
 
-  set jurisdiction (jurisdiction: string) {
-    this.setJurisdiction(jurisdiction)
+  /** Retrieve text based on selected action/flow */
+  get actionNowButtonText (): string {
+    if (this.isContinuationIn) return 'Continue In Now'
+    if (this.isAmalgamation) return 'Amalgamate Now'
+    if (this.isConversion) return 'Alter Now'
+    if (this.isRestoration) return 'Restore Now'
+    if (this.isChangeName) return 'Change Name Now'
+    return 'Incorporate Now'
```

Comment:
Should this be "Register Now"? Need to ask UI.

Reply:
According to Yui, there has been lots of back and forth regarding this and the comment above (Incorporate or Register). Apparently, Janis was against using different terminologies but business staff were adamant on making the distinction between the two.

I'll be asking Ethan on what the decision has fallen on.